Plugins/GitUIPluginInterfaces/CommitData.cs

```diff
@@ -35,5 +35,7 @@ public sealed class CommitData
         /// Gets and sets the commit message.
         /// </summary>
         public string Body { get; set; }
+
+        public string? Notes { get; set; }
```

Comment:
Please add xml-docs

Reply:
And if I may suggest something, it would be removing unnecessary braces. Single-statement blocks are ok without them. During the past 22 years of my professional programming I`ve seen one single error caused by lack of braces, and many more errors caused by code not fitting on screen vertically (and developers being too lazy to scroll and read some more context). :)

Reply:
StyleCop is used. There is a rule to remove unnecessary parenthesis. Maybe there is something similar here.

Note though that RussKie uses his widescreen monitor to have extra long lines, I want to fit at least two editors and some toolbars on my 3K monitor so we have discussions. I like it short regardless, easier to read code.
But RussKie decides.

Reply:
No, no, I was referring to extra braces, not parenthesis. 

if (b) 
{ 
    then(); 
} 

takes more VERTICAL space than 

if (b) 
    then();

Reply:
> No, no, I was referring to extra braces, not parenthesis.

That was just an example.

In this case I do not want to remove the braces though. (The start brace should have been on the previous line and no mandatory line after the end brace, but that was a decision taken by someone at one point ant it is OK.)